compiler/one-cmds/tests/one-import-onnx_ext_neg_001.test

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# test for one-import-onnx to invoke extension
+# error for both force_ext and disable_ext given
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  rm -f ../bin/one-import-onnx-ext
+  if grep -q "one-import-onnx: error" "${logfile}"; then
+    echo "${filename_ext} SUCCESS"
+    exit 0
+  fi
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+inputfile="./onnx_conv2d_conv2d.onnx"
+outputfile="./onnx_conv2d_conv2d.onnx_ext_neg_001.circle"
+logfile=${filename}.log
+
+rm -f ${outputfile}
+rm -f ${logfile}
+
+# copy dummy-compile to bin folder
+cp dummy-onnx-ext ../bin/one-import-onnx-ext
+
+# run test 
+one-import-onnx \
+--force_ext \
+--disable_ext \
```

Comment:
(Not for this PR) How about printing proper error message for this case? For example, "--force_ext and --disable_ext should not be given together".

Reply:
with https://github.com/Samsung/ONE/pull/13921/files#diff-f9bdcab801a5559dcfa3607be38b5f2061f281e40c0ebfbe87bb8872f4b622a2R185,
```
    extension_group = parser.add_argument_group('extension arguments')
    use_extension = extension_group.add_mutually_exclusive_group()
```
two options are added as exclusive.
if both is given, command will drop error message like

> one-import-onnx: error: argument --disable_ext: not allowed with argument --force_ext

which is processed in `arser` module.

Reply:
Thanks for explanation!

> which is processed in arser module.

Maybe `argparse` module?

BTW, can I ask why `--use_extension` argument is added? I can't find where it is used.

Reply:
Got some off-line comments from @mhs4670go and found that 
```
parser.add_argument('--use_extension', type=str, help=argparse.SUPPRESS)
```
isn't necessary for this case.

Reply:
> where --use_extension isn't shown

-> this is because `help=argparse.SUPPRESS` is given

> as an option but use_extension is used as some virtual option for group of [--force_ext | --disable_ext]

this is not correct.